cmd/world/root/root_test.go

```diff
@@ -134,19 +142,94 @@ func TestCreateStartStopPurge(t *testing.T) {
 	assert.Assert(t, healthResponse.IsServerRunning)
 	assert.Assert(t, healthResponse.IsGameLoopRunning)
 
+	// Restart cardinal
+	rootCmd.SetArgs([]string{"cardinal", "restart", "--detach"})
+	err = rootCmd.Execute()
+	assert.NilError(t, err)
+
+	// Check cardinal health after restart
+	resp, err = http.Get("http://127.0.0.1:3333/health")
```

Comment:
This health check code is repeated several times in this test. I'd suggest breaking it out into a helper function:

func getHealthCheck() (*healthResponse, error) {
    ...
}

This has the added benefit that the `type healthResponse struct` is only defined once.

Reply:
updated